tests/st/a2a3/tensormap_and_ringbuffer/heap_empty_ring_rebase/test_heap_empty_ring_rebase.py

```
@@ -0,0 +1,85 @@
+                "core_type": "aic",
+                "signature": [D.OUT],
+            },
+            {
+                "func_id": 1,
+                "name": "COPY_FIRST",
+                "source": "kernels/aic/kernel_copy_first.cpp",
+                "core_type": "aic",
+                "signature": [D.IN, D.INOUT],
+            },
+        ],
+    }
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)